hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestColumnStatsIndex.scala

```diff
@@ -365,6 +304,36 @@ class TestColumnStatsIndex extends HoodieClientTestBase {
     })
   }
 
+  private def doWriteAndValidateColumnStats(testCase: ColumnStatsTestCase,
```

Comment:
This method is extracted from the existing logic of reading the Json source as the input.